CGPs/cgp-0141.md

```diff
@@ -3,7 +3,7 @@ cgp: 141
 title: CeloPG H2 2024 Budget
 date-created: 2024-06-26
 author: Luuk Weber (LuukDAO) & Monty Bryant
-status: DRAFT
+status: EXECUTED
 discussions-to: https://forum.celo.org/t/celopg-h2-2024-budget/8231
 governance-proposal-id:
```

Comment:
@ericnakagawa when you update the status to `"EXECUTED"` you will need to add `governance-proposal -id` and  `date-executed `values, otherwise the bot added by @jmrossy will show errors and Celo Mondo will go down too if PR approved.

Reply:
Celo Mondo will not 'go down' but @work-0xj4an  is correct that the CI checks here enforce that executed proposals have a valid ID. Ideally they should also have a date

Reply:
Please make these changes @work-0xj4an.